test/loaders/template-loader.js

```diff
@@ -0,0 +1,22 @@
+const TemplateLoader = require("../../lib/loaders/template-loader");
+const fs = require("fs");
+const HTMLInfo = require("../../lib/models/html-info");
+require('chai').should();
+
+describe('PipeModuleLoader', function () {
+    it('should return a string with the content of a template if it was loaded earlier without reading from the file', function () {
+        const htmlInfo = new HTMLInfo(undefined, "test_template_loader_temp.txt");
```

Comment:
Why not just store the file in the repo like it's done for the other tests?

Reply:
The files is created, then removed to ensure that the template-loader is no longer reading from the file, this test is not very applicable to the current implementation of template-loader so I can remove it.